client/src/views/pages/Homepage.js

```
@@ -1,21 +1,22 @@
 import { useEffect, useState } from "react";
 import {Avatar,  Header} from 'rsuite'
-import axios from 'axios'
-import { baseURL } from "./SignUp";
+import { fetchUser } from "../../services/auth";
 
 const Homepage =  () => {
 
     const[ existingUsers, setExistingUsers] = useState([]);
-
+    
     useEffect(() => {
-        const fetchUser = async () => {
-            const activeUser = await axios.get(`${baseURL}/api/users`)
-            setExistingUsers(activeUser.data)
-            console.log(existingUsers)
-        }
-        fetchUser()
-    }, [setExistingUsers])
-    console.log(existingUsers)
+        (async () => {
+            try{
+                const userList = await fetchUser()
+                const sharedUser = userList.filter(obj => obj.shared === 1)
+                setExistingUsers(sharedUser)
+            } catch (e) {
+                setExistingUsers(false)
```

I think you should throw an error